client/ayon_core/hosts/houdini/plugins/publish/extract_mantra_ifd.py

```diff
@@ -22,8 +22,11 @@ def process(self, instance):
 
         ropnode = hou.node(instance.data.get("instance_node"))
         output = ropnode.evalParm("soho_diskfile")
-        staging_dir = os.path.dirname(output)
+        staging_dir, file_name = os.path.split(output)
         instance.data["stagingDir"] = staging_dir
+        # render rop
+        self.log.debug("Writing IFD '{}' to '{}'".format(file_name, staging_dir))
+        render_rop(ropnode)
```

Comment:
You need to clean up the lines from 32 to 41, as it errors out the extractor during publishing